doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,122 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important options of CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
+^^^^^^^^^^^^^^^^
+
+Specify a generator.
+
+The default is depending on the system.
+
+You can check the default generator and available generators by ``cmake --help``.
+
+.. code-block:: console
+
+   $ cmake --help
+   ...
+   The following generators are available on this platform (* marks default):
+     Green Hills MULTI            = Generates Green Hills MULTI files
+                                   (experimental, work-in-progress).
+   * Unix Makefiles               = Generates standard UNIX makefiles.
+     Ninja                        = Generates build.ninja files.
+     Ninja Multi-Config           = Generates build-<Config>.ninja files.
+     Watcom WMake                 = Generates Watcom WMake makefiles.
+     CodeBlocks - Ninja           = Generates CodeBlocks project files.
+     CodeBlocks - Unix Makefiles  = Generates CodeBlocks project files.
+     CodeLite - Ninja             = Generates CodeLite project files.
+     CodeLite - Unix Makefiles    = Generates CodeLite project files.
+     Eclipse CDT4 - Ninja         = Generates Eclipse CDT 4.0 project files.
+     Eclipse CDT4 - Unix Makefiles= Generates Eclipse CDT 4.0 project files.
+     Kate - Ninja                 = Generates Kate project files.
+     Kate - Unix Makefiles        = Generates Kate project files.
+     Sublime Text 2 - Ninja       = Generates Sublime Text 2 project files.
+     Sublime Text 2 - Unix Makefiles
+
+Here is an example how to specify ``Visual Studio 17 2022 x64`` as a generator.
+You can specify a target platform name (architecture) with the ``-A`` option.
+
+.. code-block:: pwsh-session
+
+  > cmake . -G "Visual Studio 17 2022" -A x64
+
+``-DCMAKE_INSTALL_PREFIX``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a directory to install Groonga.
+
+The default is depending on the system, e.g. ``/usr/local`` or ``C:/Program Files/groonga``.
+
+Here is an example how to specify ``/tmp/local/`` as an install directory.
+
+.. code-block:: console
+
+   $ cmake . -DCMAKE_INSTALL_PREFIX=/tmp/local/
```

Comment:
Why did you remove an example for Windows?
How about keeping examples for Windows and non-Windows?

Reply:
I have added examples for Windows in case that a parameter is different between UNIX and Windows.